bokehjs/src/coffee/callback/customjs.coffee

```diff
@@ -1,5 +1,5 @@
 _ = require "underscore"
-Model = require "../models/model"
+Model = require "../model/model"
```

Comment:
And if we don't put `model.coffee` in `src/coffee/models` then it should just go in `src/coffee`, not `src/coffee/model`